### PR TITLE
Small set of minor fixes (mainly typo)

### PR DESCRIPTION
**Pull request by Thannoy**
This PR is a proposal of a few very simple fixes

### CODE REVIEW

File: book/07-git-tools/sections/advanced-merging.asc [hunk 1]
```diff
@@ -115,7 +115,7 @@ $ git status -sb
 
 The `git merge --abort` option tries to revert back to your state before you ran the merge. The only cases where it may not be able to do this perfectly would be if you had unstashed, uncommitted changes in your working directory when you ran it, otherwise it should work fine.
 
-If for some reason you find yourself in a horrible state and just want to start over, you can also run `git reset --hard HEAD` or wherever you want to get back to. Remember again that this will blow away your working directory, so make sure you don't want any changes there.
```
Comment (ben) [inline book/07-git-tools/sections/advanced-merging.asc]: How about this?

> If for some reason you just want to start over, you can also run `git reset --hard HEAD`, and your repository will be back to a known state.

Comment (Thannoy) [inline book/07-git-tools/sections/advanced-merging.asc]: Yes, this is synthetic and clear. Maybe we can also say that the known state reached is the last commit of the current branch.

Whats about this updated proposal?
> If for some reason you just want to start over, you can also run git reset --hard HEAD, and your repository will be back **to last commit state. Remember again that any uncommited work will be lost,** so make sure you don't want any changes there.

Comment (ben) [inline book/07-git-tools/sections/advanced-merging.asc]: Great! One more grammar/spelling fix:

> If for some reason you just want to start over, you can also run `git reset --hard HEAD`, and your repository will be back to the last committed state. Remember that any uncommitted work will be lost, so make sure you don't want any of your changes.

Comment (Thannoy) [inline book/07-git-tools/sections/advanced-merging.asc]: Perfect. I will amend this commit.

**Comment by ben** [comment]
Nice. Thanks!

**Pull request merged**